src/redempt/redlib/region/Region.java

```diff
@@ -108,7 +109,26 @@ public abstract class Region implements Cloneable {
 	 * @return The stream of all Blocks contained in this Region
 	 */
 	public abstract Stream<Block> stream();
-	
+
+	/**
+	 * Returns threadLocalRandom
+	 * @return ThreadLocalRandomInstance
+	 */
+	public ThreadLocalRandom getLocalRandom(){
+		return ThreadLocalRandom.current();
+	}
+
+	/**
+	 * Returns random number between min (inclusive) and max (inclusive)
+	 *
+	 * @param origin Min number
+	 * @param bound  Max number
+	 * @return Int
+	 */
+	public int GetRandomNumInclusive(int origin, int bound){
```

Comment:
This has no business being a public instance method, and it shouldn't start with a capital letter. It's also so small that I don't think making a helper function for it even makes sense.

Reply:
Yea I only created it because I misunderstood one of your previous notes, my bad